src/profile-logic/call-tree.js

```diff
@@ -203,12 +204,9 @@ export class CallTree {
         icon = ExtensionIcon;
       }
 
-      const formatNumber = this._isIntegerInterval
-        ? _formatIntegerNumber
-        : _formatDecimalNumber;
+      const formattedTotalTime = formatNumber(totalTime, 2, 1);
+      const formattedSelfTime = formatNumber(selfTime, 2, 1);
```

Comment:
I think you still want to check for `_isIntegerInterval`: then you can pass "0" as second parameter if it's true. And the first parameter should always be `1`.